aiogram/dispatcher/handler.py

```diff
@@ -98,7 +102,7 @@ def unregister(self, handler):
 
         if self.middleware_key:
             try:
-                await self.dispatcher.middleware.trigger(f"pre_process_{self.middleware_key}", args + (data,))
+                await self.dispatcher.middleware.trigger(self.formatted_middleware_key, args + (data,))
```

Comment:
Don't change this place.
This change has no positive effect.

Reply:
but what's the point of a f-string here